src/components/Proposal/View/Budget/CompetingProposalsSidebar.css

```diff
@@ -0,0 +1,138 @@
+.ui.push.right.very.wide.visible.sidebar.CompetingProposalsSidebar {
+  width: 600px;
```

Comment:
What about mobile?

Reply:
We don't have the mobile design yet, I'll ask for it. In the meantime, this is not available in prod so it shouldn't matter.

Reply:
Feature is disabled in Desktop, I created another issue (#846) for this, to reduce the scope of this PR